exercises/06-rock-paper-scissors.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Rock Paper Scissors</title>
+</head>
+<body>
+  <button onclick="
+    userChoice = 'rock';
+    playTheGame();
+    "
+  >Rock</button>
+
+  <button onclick="
+    userChoice = 'paper';
+    playTheGame();
+"
+  >Paper</button>
+
+  <button onclick="
+    userChoice = 'scissors';
+    playTheGame();
+"
+  >Scissors</button>
+
+  <script>
+    const choices = ['rock', 'paper', 'scissors']
+    let computerChoice; 
+    let userChoice;
+    getComputerChoice = () => {
+      randomChoice = choices[Math.floor(Math.random() * choices.length)];
+      return randomChoice;
+    }
+
+    getResult = () => {
```

Comment:
Also, something to keep in mind: 
This approach is good for the beginning but is not acceptable for scalability.

Reply:
No need to change, but just a suggestion for next assignments. 
try to think of scalability and dynamic approach. 

if we are looking at the problem with the code structure in mind.
you can structure your choices in a way where the solution will not depend on hard-coded choice names but on their defined positions. 
for example 

```JS
[ 'scissors', 'rock', 'paper']
// rock beats scissors -> 0 < 1
// paper beats rock -> 1 < 2
// scissors beats paper -> because paper is the last in the list -> the first in the list beats it by default
```